cli/roles.go

```diff
@@ -1,27 +1,60 @@
 package main
 
 import (
-	"fmt"
-
-	"github.com/riotgames/key-conjurer/providers"
+	"github.com/RobotsAndPencils/go-saml"
```

Comment:
This is an unsupported and archived project, are there any better solutions?

Reply:
Check https://github.com/RobotsAndPencils/go-saml. It mentions a couple of alternatives